mmdet/integration/nncf/compression.py

```diff
@@ -61,6 +64,30 @@ def is_checkpoint_nncf(path):
     except FileNotFoundError:
         return False
 
+def get_nncf_config_from_meta(path):
+    logger = get_root_logger()
+    checkpoint = torch.load(path)
+    meta = checkpoint.get('meta', {})
+
+    nncf_enable_compression = meta.get('nncf_enable_compression', False)
+    assert nncf_enable_compression, \
+            'get_nncf_config_from_meta should be run for NNCF-compressed checkpoints only'
+
+    config_text = meta['config']
+    with tempfile.NamedTemporaryFile(prefix='config_', suffix='.py', mode='w') as f_tmp:
+        f_tmp.write(config_text)
+        cfg = mmcv.Config.fromfile(f_tmp.name)
```

Comment:
I guess it's used for debug purposes only. If so, I'd suggest adding a flag to make it only when needed.

Reply:
No. It is really required every time when we try to export an NNCF-compressed model
Otherwise export won't work

Reply:
Does that mean, that you can't compose config directly from the `config_text` and have nothing to do but dump it to a file and load back?

Reply:
Yes, AFAIK it is impossible in the current mmcv -- see the implementation of `mmcv.Config`-s constructor in the file
mmcv/utils/config.py:
https://github.com/open-mmlab/mmcv/blob/master/mmcv/utils/config.py#L268
(cfg_text param of `__init__` is used only for text representation)

Reply:
Ok, then...